integration_test/networks.test.ts

```diff
@@ -0,0 +1,485 @@
+/*
+ * Copyright 2021, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict'
+
+import { expect } from 'chai'
+import { providers } from 'ethers'
+import { instance, mock, when } from 'ts-mockito'
+
+import { getL1Network } from '../src'
+import {
+  addCustomNetwork,
+  getL2Network,
+  isL1Network,
+  L1Network,
+  l1Networks,
+  L2Network,
+  l2Networks,
+} from '../src/lib/dataEntities/networks'
+import { SEVEN_DAYS_IN_SECONDS } from '../src/lib/dataEntities/constants'
+
+/**
+ * This is a unit test rather than an integration test.
+ * TODO -> We'll move it out of integration_test/ when we restructure the tests
```

Comment:
I've generally tried to avoid leaving any TODO in the SDK and instead track them via tickets. I find TODOs in the code get forgotten about. Happy to chat more about this though